content/tracing/languages/java.md

```diff
@@ -0,0 +1,274 @@
+
+Priority sampling ensures that distributed traces are complete by assigning and propagating a priority value along each trace.
+
+Current Priority Values (more may be added in the future):
+
+|Sampling Value | Effect                                                                                                     |
+|---------------|:----------------------------------------------------------------------------------------------------------|
+|SAMPLER_DROP   | The sampler automatically decided to not keep the trace. The Agent will drop it.                           |
+|SAMPLER_KEEP   | The sampler automatically decided to keep the trace. The Agent will keep it. Might be sampled server-side. |
+|USER_DROP      | The user asked to not keep the trace. The Agent will drop it.                                              |
+|USER_KEEP      | The user asked to keep the trace. The Agent will keep it. The server will keep it too.                     |
+
+
+Priority sampling is disabled by default. To enable it, configure the `priority.sampling` flag to `true` ([see how to configure the client above](#configuration)).
+
+Once enabled, the sampler automatically assigns a priority to traces, depending on their service and volume.
+
+You can also set this priority manually to either drop a non-interesting trace or to keep an important one.
+```java
+import datadog.opentracing.DDSpan;
+import datadog.trace.api.Trace;
+import datadog.trace.common.sampling.PrioritySampling;
+import io.opentracing.util.GlobalTracer;
+
+public class MyClass {
+    @Trace
+    public static void myMethod() {
+        // grab the active span out of the traced method
+        DDSpan ddspan = (DDSpan) GlobalTracer.get().activeSpan();
+        // ask the sampler to keep the current trace
+        ddspan.setSamplingPriority(PrioritySampling.USER_KEEP);
+        // method impl follows
+    }
+}
+```
+
+## Integrations
+
+### Web Frameworks
+
+`dd-java-agent` includes support for automatically tracing the following web frameworks.
+
+| Server | Versions |
+|:------------- |:-------------|
+| Java Servlet Compatible | 2.3+, 3.0+ |
+
+*Note:* Many application servers are Servlet compatible, such as Tomcat, Jetty, Websphere, Weblogic, etc.
+Also, frameworks like Spring Boot and Dropwizard inherently work because they use a Servlet compatible embedded application server.
+
+Don't see your desired networking framework? We're continually adding additional support, [check with our team](/help) to see if we can help.
+
+### Networking Frameworks
+
+`dd-java-agent` includes support for automatically tracing the following networking frameworks.
+
+| Framework      | Versions           |
+|:-------------|:-------------|
+| [OkHTTP](https://github.com/opentracing-contrib/java-okhttp) | 3.x |
+| [Apache HTTP Client](https://github.com/opentracing-contrib/java-apache-httpclient) | 4.3 + |
+| [AWS SDK](https://github.com/opentracing-contrib/java-aws-sdk) | 1.11.0+ |
```

Comment:
We need to remove this for Thursday.  AWS SDK support will not be available until next week @l0k0ms

Reply:
👍